### PR TITLE
[Fix-bug] LPS-89051

### PR DESCRIPTION
**Pull request by yuhai**
@dantewang, the pr resolved the test error from https://github.com/liferay-core-infra/liferay-portal/pull/2148#issuecomment-1286049427

### CODE REVIEW

**Comment by yuhai** [comment]
@zhangyan0701, could you please check the test result after the test finished? Thanks.

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

File: modules/apps/portal-store/portal-store-s3-test/src/testIntegration/java/com/liferay/portal/store/s3/test/AbortedMultipartUploadCleanerTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Hai Yu
+ */
+@RunWith(Arquillian.class)
+public class AbortedMultipartUploadCleanerTest {
```
Comment (tinatian) [inline modules/apps/portal-store/portal-store-s3-test/src/testIntegration/java/com/liferay/portal/store/s3/test/AbortedMultipartUploadCleanerTest.java]: @yuhai I do not think we need this test, just modify directly is fine, as it is obvious a typo

File: modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleanerMessageListener.java [hunk 2]
```diff
@@ -70,6 +71,12 @@ protected void deactivate() {
 
 	@Override
 	protected void doReceive(Message message) throws Exception {
+		String storeClassName = PropsValues.DL_STORE_IMPL;
```
Comment (tinatian) [inline modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleanerMessageListener.java:76]: @yuhai Please use portal profile, do not enable this component when dl store is not set, like what you did for the company log servlet.

**Comment by tinatian** [comment]
@yuhai Please see my comment and resend.

Thanks.
Tina

**Pull request closed without merging**